src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -68,9 +84,8 @@ export class PassFinalisationPage extends PracticeableBasePageComponent {
   passCertificateCtrl: string = 'passCertificateNumberCtrl';
   @ViewChild('passCertificateNumberInput')
   passCertificateNumberInput: ElementRef;
-
   inputSubscriptions: Subscription[] = [];
-
+  testOutcome: string = '1';
```

Comment:
can't see a use for this

Reply:
The radio buttons being displayed and validation from the office page are driven by the outcome.

Reply:
There should be at least an enum somewhere for those test outcomes/activity codes - right?